classes/TranslatableBehavior.php

```diff
@@ -58,15 +59,25 @@ public function __construct($model)
 
         $this->initTranslatableContext();
 
-        $this->model->bindEvent('model.beforeGetAttribute', function($key) {
-            if ($key !== 'translatable' && $this->isTranslatable($key)) {
-                return $this->getAttributeTranslated($key);
+        $this->model->bindEvent('model.beforeGetAttribute', function($key) use ($model) {
```

Comment:
@mjauvin Space between `function` and `(` please :)
```suggestion
        $this->model->bindEvent('model.beforeGetAttribute', function ($key) use ($model) {
```